app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md

```diff
@@ -52,7 +52,7 @@ Some teacher training courses in England are salaried – this means you [work a
 
 ### To apply for your visa, follow these steps 
 
-Visit [Find postgraduate teacher training](https://www.gov.uk/find-postgraduate-teacher-training-courses) and filter your search results by visa sponsorship. 
+Visit [Find postgraduate teacher training](https://find-teacher-training-courses.service.gov.uk/) and filter your search results by visa sponsorship. 
```

Comment:
Is it worth improving this link text now as well? Referring to a service name is not best practice (also - this isn't the service name any more!)